redaxo/src/core/lib/form/form_base.php

```diff
@@ -988,6 +988,8 @@ public function setMessage($message)
     public function getMessage()
     {
         $message = rex_request($this->getName() . '_msg', 'string');
+        $message = strip_tags($message, '<span><p><div>');
```

Comment:
ich frage mich wie das bisher genutzt wird und wie man es am besten einschränkt ohne bestehende funktionalität zu gefährden.

der fix hier würde verhindern dass man `<script>` tags einfügen kann, aber ich denke es wäre dennoch möglich z.B. `<span onclick="..">` o.ä. zu injecten.

Reply:
striptags mit zweitem Parameter ist wohl nicht safe: https://externals.io/message/102394#102462

Wir können unser AntiXSS-Vendor hier nutzen. Oder so wie in i18n vorgehen, wo wir auch nur bestimmte Tags zulassen. Allerdings dann auch nur ohne weitere Attribute etc. Ich weiß nicht, was hier so vorkommt.

Reply:
https://github.com/redaxo/redaxo/blob/master/redaxo/src/core/lib/util/i18n.php#L267-L268

Reply:
guter hinweis.. hab mal ne neue rex_string methode angelegt